crates/bevy_math/src/rects/urect.rs

```diff
@@ -284,17 +287,21 @@ impl URect {
     ///
     /// ```rust
     /// # use bevy_math::{URect, UVec2};
-    /// let r = URect::new(2, 2, 4, 4); // w=2 h=2
+    /// let r = URect::new(4, 4, 6, 6); // w=2 h=2
     /// let r2 = r.inset(1); // w=3 h=3
-    /// assert_eq!(r2.min, UVec2::splat(1));
-    /// assert_eq!(r2.max, UVec2::splat(5));
+    /// assert_eq!(r2.min, UVec2::splat(3));
```

Comment:
Line above (GitHub won't let me comment) has wrong size in comment, should be `w=4 h=4`.